src/anyon/anyon.jl

```diff
@@ -23,28 +23,49 @@ Quantum Processing Unit:
 struct AnyonYukonQPU <: AbstractQPU
     client                  ::Client
     status_request_throttle ::Function
+    connectivity            ::AbstractConnectivity
```

Comment:
The type of Yukon's Connectivity should be `LineConnectivity` as Yukon will never have any other type.